src/uv.cc

```diff
@@ -70,9 +70,10 @@ void ErrName(const FunctionCallbackInfo<Value>& args) {
         "DEP0119").IsNothing())
     return;
   }
-  int err;
-  if (!args[0]->Int32Value(env->context()).To(&err)) return;
-  CHECK_LT(err, 0);
+  if (args[0]->IsNullOrUndefined()) {
+    return args.GetReturnValue().SetUndefined();
+  }
+  const int err = args[0].As<v8::Int32>()->Value();
   const char* name = uv_err_name(err);
```

Comment:
If this function is exposed to userland, I'd find it would be safer to use [`uv_err_name_r`](http://docs.libuv.org/en/v1.x/errors.html?highlight=uv_err_name#c.uv_err_name_r) instead here.

Reply:
I will mark this as resolved by https://github.com/nodejs/node/pull/44421; thanks for the suggestion <3